concepticondata/conceptlists/Gallatin-1836-180.tsv

```diff
@@ -0,0 +1,181 @@
+Gallatin-1836-180-67	67	Summer	1342	SUMMER
+Gallatin-1836-180-68	68	Autumn	1276	AUTUMN
+Gallatin-1836-180-69	69	Winter	1471	WINTER
+Gallatin-1836-180-70	70	Wind	960	WIND
+Gallatin-1836-180-71	71	Lightning	1654	LIGHTNING
+Gallatin-1836-180-72	72	Thunder	1150	THUNDER
+Gallatin-1836-180-73	73	Rain	658	RAIN (PRECIPITATION)
+Gallatin-1836-180-74	74	Snow	784	SNOW
+Gallatin-1836-180-75	75	Hail	609	HAIL
+Gallatin-1836-180-76	76	Fire	221	FIRE
+Gallatin-1836-180-77	77	Water	948	WATER
+Gallatin-1836-180-78	78	Ice	617	ICE
+Gallatin-1836-180-79	79	Earth, Land	3603	EARTH OR LAND
+Gallatin-1836-180-80	80	Sea	1474	SEA
+Gallatin-1836-180-81	81	River	666	RIVER
+Gallatin-1836-180-82	82	Lake	624	LAKE
+Gallatin-1836-180-83	83	Valley	921	VALLEY
+Gallatin-1836-180-84	84	Hill	614	HILL
+Gallatin-1836-180-85	85	Mountain	639	MOUNTAIN
+Gallatin-1836-180-86	86	Island	622	ISLAND
+Gallatin-1836-180-87	87	Stone, Rock	2125	STONE OR ROCK
+Gallatin-1836-180-88	88	Copper	1825	COPPER
+Gallatin-1836-180-89	89	Iron	621	IRON
+Gallatin-1836-180-90	90	Maize	506	MAIZE
+Gallatin-1836-180-91	91	Tree	906	TREE
+Gallatin-1836-180-92	92	Wood	1803	WOOD
+Gallatin-1836-180-93	93	Leaf	628	LEAF
+Gallatin-1836-180-94	94	Bark	1204	BARK
+Gallatin-1836-180-95	95	Grass	606	GRASS
+Gallatin-1836-180-96	96	Oak	644	OAK
+Gallatin-1836-180-97	97	Pine-tree	1740	PINE
+Gallatin-1836-180-98	98	Flesh, Meat	2615	FLESH OR MEAT
+Gallatin-1836-180-99	99	Beaver	1194	BEAVER
+Gallatin-1836-180-100	100	Deer	1936	DEER
+Gallatin-1836-180-101	101	Bison, Buffalo	3142	BISON
+Gallatin-1836-180-102	102	Bear	1820	BEAR
+Gallatin-1836-180-103	103	Wolf	522	WOLF
+Gallatin-1836-180-104	104	Dog	2009	DOG
+Gallatin-1836-180-105	105	Fox	1312	FOX
+Gallatin-1836-180-106	106	Squirrel	1628	SQUIRREL
+Gallatin-1836-180-107	107	Rabbirt, Hare	2345	LEPORID (RABBIT OR HARE)
+Gallatin-1836-180-108	108	Snake	730	SNAKE
+Gallatin-1836-180-109	109	Bird	937	BIRD
+Gallatin-1836-180-110	110	Egg	744	EGG
+Gallatin-1836-180-111	111	Goose	1187	GOOSE
+Gallatin-1836-180-112	112	Duck	1360	DUCK
+Gallatin-1836-180-113	113	Pigeon	1853	DOVE
+Gallatin-1836-180-114	114	Partridge	3681	PARTRIDGE
+Gallatin-1836-180-115	115	Turkey	2386	TURKEY
+Gallatin-1836-180-116	116	Fish	227	FISH
+Gallatin-1836-180-117	117	White	1335	WHITE
+Gallatin-1836-180-118	118	Black	163	BLACK
+Gallatin-1836-180-119	119	Red	156	RED
+Gallatin-1836-180-120	120	Blue	837	BLUE
+Gallatin-1836-180-121	121	Yellow	1424	YELLOW
+Gallatin-1836-180-122	122	Green	1425	GREEN
+Gallatin-1836-180-123	123	Great, Big	1202	BIG
+Gallatin-1836-180-124	124	Small, Little	1246	SMALL
+Gallatin-1836-180-125	125	Strong	785	STRONG
+Gallatin-1836-180-126	126	Old	1229	OLD
+Gallatin-1836-180-127	127	Young	1207	YOUNG
+Gallatin-1836-180-128	128	Good	1035	GOOD
+Gallatin-1836-180-129	129	Bad	1292	BAD
+Gallatin-1836-180-130	130	Handsome	3814	HANDSOME
+Gallatin-1836-180-131	131	Ugly	1548	UGLY
+Gallatin-1836-180-132	132	Alive, Life	3184	BE ALIVE OR LIFE
+Gallatin-1836-180-133	133	Dead, Death		
```

Comment:
What about BE DEAD OR DEATH OR DIE? I think we had the discussion recently whether all three concepts must be colexified/lumped together in a list to justify mapping to this set (or whether two is enough), but I don't remember the outcome...

Reply:
Yeah, that was in Williamson-1973, where we decided to unmap. I'll leave it unmapped here for now as well but will revisit this concept class later on. Thank you!